src/app/(sidebar)/pitch-deck/page.tsx

```
@@ -0,0 +1,129 @@
+export default function PitchDeckPage() {
+  const breadcrumbItems = [
+    { name: "Home", url: "https://blawby.com" },
+    { name: "Blawby Investor Pitch", url: "https://blawby.com/pitch-deck" },
+  ];
+  const breadcrumbSchema = getBreadcrumbSchema(breadcrumbItems);
+
+  return (
+    <SidebarLayoutContent breadcrumbs={null}>
+      <script
+        type="application/ld+json"
+        dangerouslySetInnerHTML={{ __html: JSON.stringify(breadcrumbSchema) }}
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)